src/index.js

```diff
@@ -1,5 +1,5 @@
 import ReactDOM from 'react-dom';
-import routes from "./routes";
+import routes from "./Routes";
```

Comment:
Rename the file not just import

Reply:
The ones still in lower case (except for register.css) are boilerplates from create-react-app.
Do you also want me to change them?
* index.css
* index.js
* index.html
* routes.js
* setupTests.js

Reply:
routes can become Routes. This convention is for components only. Other files are just javascript files, not components. So let them be the way they are.

Reply:
ok, so I only need to change the Routes (and the Register), is this correct?

Reply:
will this mean I also need to change line 2 and 5 in index.js to 
`import **Routes** from "./Routes";

- - -

`**Routes**, document.getElementById('root')

?

Reply:
should Routes actually be a `.jsx` file, instead of `.js`?

Reply:
Done. just made the changes from the terminal. will start relying on terminal instead of vscode gui from now on 🤪

<img width="375" alt="Screen Shot 2020-06-15 at 4 38 12 pm" src="https://user-images.githubusercontent.com/29667122/84625528-c495ef80-af26-11ea-8f98-e8311721e994.png">
<img width="349" alt="Screen Shot 2020-06-15 at 4 38 20 pm" src="https://user-images.githubusercontent.com/29667122/84625539-c8297680-af26-11ea-8730-022b9293097b.png">